docs/docs-source/docs/modules/get-started/pages/install-cloudflow.adoc

```diff
@@ -0,0 +1,322 @@
+              kubectl patch MutatingWebhookConfiguration $HOOK_NAME -n cloudflow -p "$JSON"
+EOF2
+----
+
+Now we can apply the changes to the webhook.
+
+  kubectl apply -f spark-mutating-webhook.yaml --namespace cloudflow
+
+That completes the installation of the Spark operator. 
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS      RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running     0          3m29s
+cloudflow-patch-spark-mutatingwebhookconfig-66xxb   0/1     Completed   0          21s
+spark-operator-sparkoperator-56d6ffc8cd-cln67       1/1     Running     0          3m29s
+----
+
+== Installing the Flink operator
+
+If you plan to write applications that utilize Flink you will need to install the Flink operator before deploying your Cloudflow application. As we can see below, the CLI will not allow you to deploy an application that does not have all dependencies installed.
+
+  $ kubectl cloudflow deploy cloudflow/examples/taxi-ride/target/taxi-ride-fare.json
+  [Error] cannot detect that Flink is installed, please install Flink before continuing (exit status 1)
+
+The Flink operator used is a version currently maintained by Lightbend and is the only Cloudflow compatible version.
+
+Install the Flink operator using the following Helm command:
+
+----
+helm install flink-operator \
+https://github.com/lightbend/flink-operator/releases/download/v0.8.2/flink-operator-0.8.2.tgz \
+--namespace cloudflow --set operatorImageName="lightbend/flinkk8soperator" --set \
+operatorVersion="v0.5.0"
+----
+
+This completes the installation of the Flink operator. 
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS      RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running     0          3m29s
+cloudflow-patch-spark-mutatingwebhookconfig-66xxb   0/1     Completed   0          21s
+spark-operator-sparkoperator-56d6ffc8cd-cln67       1/1     Running     0          2m29s
+flink-operator-7999fdd879-phqlg                     1/1     Running     0          1m29s
+----
+
+== Installing Enterprise components
+If you have a commercial agreement with Lightbend, you are eligible to install the Cloudflow Enterprise components. 
+
+Before installing, you need to make sure you have the credentials required to download the Helm chart from the Lightbend commercial repository.
+
+Add the Lightbend Helm repository.
+
+  helm repo add lightbend https://repo.lightbend.com/helm-charts/
+
+Update Helm to make sure all charts are up-to-date
+
+  helm repo update
+
+Install the components, note that you set the `$USERNAME` and `$PASSWORD` environment variables before executing this command. 
+
+We recommend using the following command to create an environment variable for username and password. 
+*This is to avoid storing this information in the shell's history file.*
+
+  read -a USERNAME
```

Comment:
This does not work on Mac:
```
read -a USERNAME
read: bad option: -a
```

Reply:
Have replaced this to just `read <VARIABLE>`